IHP/Controller/Render.hs

```diff
@@ -100,7 +101,21 @@ renderJson' additionalHeaders json = respondAndExit $ responseLBS status200 ([(h
 --
 renderNotFound :: (?context :: ControllerContext) => IO ()
 renderNotFound = do
```

Comment:
This is the same as  `IHP.Controller.NotFound.renderNotFound`, can we remove this one in `IHP.Controller.Render`?